### ISSUE TITLE
Sophia tutorials are outdated

### ISSUE THREAD

**Issue by mradkov** [aeternity/tutorials#61]
Most of the contracts are using the old Sophia syntax.
The section with fungible tokens is not using the [AEX-9 standard for fungible tokens](https://github.com/aeternity/AEXs/blob/master/AEXS/aex-9.md) on aeternity, which will mislead new developers in the ecosystem and result in various token implementations, not supported by wallets.

**Comment by thepiwo** [comment]
Yes, I'd say we should remove those and eventually replace with AEX-9 once finalized

**Comment by mradkov** [comment]
> Yes, I'd say we should remove those and eventually replace with AEX-9 once finalized

Agree!

What we could least do for now is to show an **outdated** sign to just warn people.
